packages/core/src/client.ts

```diff
@@ -355,6 +355,10 @@ interface RequestCacheEntry {
 }
 
 /**
+ *  <head>
```

Comment:
Does it have to be at the top of the comment? Ideally it'd be as low as possible